analyses/chromosomal-instability/00-setup-breakpoint-data.R

```diff
@@ -92,6 +103,18 @@ option_list <- list(
     'OpenPBTA-analysis') that specifies the BED regions file that indicates the 
     effectively surveyed regions of the genome for the WXS samples.",
     metavar = "character"
+  ), 
+  make_option(
+    opt_str = "--gap", default = 0,
```

Comment:
Will this catch adjacent breaks, like a break at position 10 and another at 11? Because those should really be the same, but I can't tell if they are considered gap of 0 or 1.

Reply:
From GenomicRanges docs: `The gap between 2 ranges is the number of positions that separate them. The gap between 2 adjacent ranges is 0.` I interpret this to mean that `maxgap = 0`, means they are right next to each other. Is this what we want?